src/test/java/com/restassured/exercise/StudentsTests.java

```diff
@@ -15,7 +15,7 @@ public void testStudentCreated() {
 	    
 		JSONObject request = new JSONObject();
 
-		request.put("name", "Estereo");
+		request.put("name", "JaimeA");
```

Comment:
Can we have this parameter pseudo-randomly, so don't have to change them constantly